wanjiku-carol/users_api
# Feature: Create Flask App and Basic Setup

**What does this PR do?**
It creates the flask app and sample routes as well as adding on Docker, Redis for caching and Postgres and their configurations.

**Describe the work done in this PR**
- Created app.py with the flask app and sample routes to be populated accordingly
- Created a models.py with a sample User model.
- Created the Dockerfile and docker-compose components to set up the container and relevant images.
- Created the env configurations for Flask, Redis , Docker and Postgres.

**How can this be tested?**

To run the docker container:

> docker-compose -up -d --build

To check the running images

> docker-ps

To run the flask app:

> python app.py 

**Screenshots**
<img width="727" alt="Screen Shot 2022-09-30 at 6 07 30 AM" src="https://user-images.githubusercontent.com/27045854/193181887-70a0de45-e224-49d7-94b0-1835f99c8e5c.png">
<img width="732" alt="Screen Shot 2022-09-30 at 6 06 55 AM" src="https://user-images.githubusercontent.com/27045854/193181909-3823d0e3-239e-4907-b43f-c1c18e655ed7.png">
